.github/workflows/scripts/generate_index.php

```diff
@@ -15,19 +15,16 @@ function update_readme($categories) {
         $category = ucwords($category);
         fwrite($readme, "\n## $category\n");
         // Sort templates alphabetically within each category
-        usort($templates, function($a, $b) { return strcmp($a['name'], $b['name']); });
+        usort($templates, function($a, $b) { return strcmp($a['template_details']['card-title'], $b['template_details']['card-title']); });
```

Comment:
@sanjacornelius There are some pretty deeply nested array paths here. Should we add some checks to make sure the expected keys actually exist?